spec.html

```diff
@@ -6445,23 +6438,23 @@ <h1>DeleteBinding ( _N_ )</h1>
 
         <emu-clause id="sec-declarative-environment-records-hasthisbinding">
           <h1>HasThisBinding ( )</h1>
-          <p>Regular declarative Environment Records do not provide a `this` binding.</p>
+          <p>The HasThisBinding concrete method of a declarative Environment Record _envRec_ takes no arguments. Regular declarative Environment Records do not provide a `this` binding. It performs the following steps when called:</p>
```

Comment:
Fine with me. And I'm taking the occasion to reword them. In particular, the word "regular" is unclear.
- For a *declarative* ER, it presumably means one that is neither a function ER or a module ER (since these subclasses do indeed override these methods).
- For an *object* ER, it can't have a corresponding meaning, because object ER doesn't have subclasses. I suspect it's a copy-pasta.